drip/utils.py

```diff
@@ -78,19 +82,23 @@ def get_fields(Model,
         out_fields.append([full_field, field_name, Model, field.__class__])
 
         if not stop_recursion and \
-                (isinstance(field, ForeignKey) or isinstance(field, OneToOneField) or \
-                isinstance(field, RelatedObject) or isinstance(field, ManyToManyField)):
+                (isinstance(field, ForeignKey) or
+                 isinstance(field, OneToOneField) or
+                 isinstance(field, RelatedObject) or
+                 isinstance(field, ManyToManyField)):
 
             if isinstance(field, RelatedObject):
                 RelModel = field.model
-                #field_names.extend(get_fields(RelModel, full_field, True))
+                # field_names.extend(get_fields(RelModel, full_field, True))
```

Comment:
We should remove this commented code line.

Reply:
I just applied flake8 in this PR. @kaozdl maybe knows if that commented line is there for a good reason